### ISSUE TITLE
Resizing / reshaping custom outline areas is twitchy

### ISSUE THREAD

**Issue by mojodna** [hotosm/osm-analytics#68]
Under some circumstances it's impossible to drag the control points, under others they can be dragged, and under yet others dragging is very jerky.

It seems likely that the histogram is trying to update while the outline area is being re-shaped rather than once it settles into a new position.

**Comment by tyrasd** [comment]
Yeah, it is indeed a bit buggy. I believe it's caused by the mixed usage of Leaflet+Leaflet.draw together with mapbox-gl-js, which doesn't always work perfectly together.

It should now be possible to refactor out Lealfet by replacing the "drawing" component with https://github.com/mapbox/mapbox-gl-draw.

**Comment by PaulUithol** [comment]
@krkwesiga @benjaminug can we add this to your first OSMA milestone please?

**Comment by jjiglesiasg** [comment]
If the Area in square KMS of the Custom area selected could be depicted it will be very Handy
